### PR TITLE
Fixed #32675 -- Doc'd that autodector changes might cause generation of no-op migrations.

### PR DESCRIPTION
**Pull request by David-Wobrock**
Refs https://code.djangoproject.com/ticket/32675

### CODE REVIEW

File: docs/releases/4.0.txt [hunk 1]
```diff
@@ -217,7 +217,9 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* The ``makemigrations`` command might generate a no-op `AlterField` operation
```
Comment (David-Wobrock) [inline docs/releases/4.0.txt:220]: I didn't know how exhaustive the release note should be about the change.
Feel free toreword or add to details to the message :)

File: docs/releases/4.0.txt [hunk 2]
```diff
@@ -217,7 +217,9 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* The ``makemigrations`` command might generate a no-op ``AlterField``
+  operation in some rare cases.
```
Comment (charettes) [inline docs/releases/4.0.txt:221]: Thanks for that! I guess we could mention that this should only happen for `ForeignKey` fields unless we're aware it happens for other options.

**Review by felixxm — COMMENTED** [review]
@David-Wobrock Thanks :+1:

**Comment by David-Wobrock** [comment]
Thanks for the quick reviews @charettes and @felixxm 💪

**Comment by felixxm** [comment]
@David-Wobrock Thanks for updates :+1: I found one more case of `to_field` in ` django.contrib.redirects`.

**Comment by David-Wobrock** [comment]
> 
> 
> @David-Wobrock Thanks for updates 👍 I found one more case of `to_field` in ` django.contrib.redirects`.

Good catch, thanks a lot! 🚀

**Pull request merged**